device/core/devdoc/device_client_requirements.md

```diff
@@ -250,6 +248,8 @@ interface DeviceMethodEventHandler {
 
 **SRS_NODE_DEVICE_CLIENT_16_086: [** Any operation (such as `sendEvent` or `onDeviceMethod`) happening after a `setRetryPolicy` call should use the policy set during that call. **]**
```

Comment:
>Any operation [](start = 37, length = 13)

Out of scope, but we should have an explicit list of operations  so we can verify test coverage.

Reply:
yeah I've added this to the twin because it's simple but it would become a little jarring here. 

---
In reply to: [171947322](https://github.com/pierreca/azure-iot-sdk-node/pull/2#discussion_r171947322) [](ancestors = 171947322)